### PR TITLE
[ADD][9.0] Module 'product_supplierinfo_variant'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
# 
# Product Supplerinfo Variant

This module displays the Product Variant in Supplier Pricelists

### CODE REVIEW

File: product_supplierinfo_variant/README.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+===========================
+Product Supplerinfo Variant
+===========================
+
+This module displays the Product Variant in Supplier Pricelists
```
Comment (elicoidal) [inline product_supplierinfo_variant/README.rst:9]: Add a period at the end

File: product_supplierinfo_variant/__init__.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline product_supplierinfo_variant/__init__.py:1]: in principle no need the copyright.
Do we need the file?

**Comment by elicoidal** [comment]
:+1:

**Comment by pedrobaeza** [comment]
You have to preserve commit history following https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0

File: product_supplierinfo_variant/__init__.py (hunk 2 above)
Comment (JordiBForgeFlow) [inline product_supplierinfo_variant/__init__.py:1]: This file is always needed in Python modules.

**Comment by JordiBForgeFlow** [comment]
@pedrobaeza But this module is brand new. What do you mean about commit history?

**Comment by pedrobaeza** [comment]
It's not new, and now I see that this is not the best repository: https://github.com/OCA/product-variant/tree/8.0/product_variant_supplierinfo

**Comment by JordiBForgeFlow** [comment]
OK! Will move and close this one.

**Comment by gurneyalex** [comment]
closing -> this is in OCA/product-variant.

**Pull request closed without merging**